src/masoniteorm/models/Model.py

```diff
@@ -563,6 +563,21 @@ def to_json(self):
         """
         return json.dumps(self.serialize())
 
+    @classmethod
+    def first_or(cls, wheres, callback=None, *args):
+        """Execute the query and get the first result or call a callback.
+
+        Returns:
+            Record|Mixed
+        """
+        self = cls()
+        self._check_is_callable(callback)
```

Comment:
what is this doing? You built a check for is callable but not using the return value form it anywhere

Reply:
@JarriqTheTechie any update on this? This is wrong right?